test/auto/injectorSpec.js

```diff
@@ -192,7 +192,7 @@ describe('injector', function() {
 
 
     it('should create $inject', function() {
-      var extraParans = angular.noop;
+      var extraParams = angular.noop;
```

Comment:
It took me a long time to try to decide if this should be `extraParams` or `extraParens`, the initial commit was not remotely helpful.